packages/loot-core/migrations/1699454123000_reports.sql

```diff
@@ -0,0 +1,27 @@
+BEGIN TRANSACTION;
+
+CREATE TABLE reports
+  (id TEXT PRIMARY KEY,
+   name TEXT,
+   start TEXT,
+   end TEXT, 
+   mode TEXT DEFAULT 'total',
+   groupBy TEXT DEFAULT 'Category',
+   balanceType TEXT DEFAULT 'Expense',
+   interval TEXT DEFAULT 'Monthly',
+   empty INTEGER DEFAULT 0,
+   hidden INTEGER DEFAULT 0,
+   uncat INTEGER DEFAULT 0,
+   selectedCategories TEXT,
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏do we even need this? We already have `conditions_op` that can be used to filter specific categories. So perhaps we can remove the duplicate DB column?

In the UI we can keep it. But when the user selects to hide a few categories - it just adds a new filter to the view using `conditions_op`.

WDYT?

Reply:
I see your point. Implementing it in my mind seems complex. Maybe there's a better way to do it.

Eg. if you unselect one category you could use "is not" & categoryId. If you unselect all and just choose one you could use "is" & categoryId. When you start adding more and more to either side eventually they meet in the middle. 

Or maybe you always use "is any" for field and op. I guess that could work, just thinking outload. Thoughts?

Reply:
Although it seems like it may be difficult to differentiate actual filters from "selected list". We could always make a new type that combines the two in one array.